src/service/entities.ts

```diff
@@ -53,10 +53,10 @@ export class DomainDefinition extends EntityDefinition {
         return [createError('Type must be string')];
       }
       if (typeof val.value !== 'string') {
-        return [''];
+        return [];
```

Comment:
Put at least stub error messages here, as this will be interpreted as no error later.